src/box/tuple.h

```diff
@@ -320,9 +320,27 @@ box_tuple_next(box_tuple_iterator_t *it);
 box_tuple_t *
 box_tuple_new(box_tuple_format_t *format, const char *data, const char *end);
 
+/**
+ * Update tuple data in a space.
+ * @param tuple  Tuple to update.
+ * @param expr
+ * @param expr_end
+ *
+ * @retval  NULL The tuple was not updated.
+ * @retval  box_tuple_t The tuple successfully updated in a space.
+ */
```

Comment:
`box_update()` is about updating a tuple in a space. This function just update a tuple.

`expr` and `expr_end` description could be grabbed from `box_update()` or `xrow_update_read_ops()`.

Nits:

* I'm a markdown guy and I like when paragraphs are separated by an empty line. IOW, I would separate the brief description from the parameters description:

  ```c
  /**
   * Blah Blah.
   *
   * @param <...>
   * <...>
   */
  ```
* Why two whitespaces?

Reply:
> I'm a markdown guy and I like when paragraphs are separated by an empty line. 

I would left as is to keep consistent with other comments, at least in this file

> Why two whitespaces?

removed

Reply:
> > I'm a markdown guy and I like when paragraphs are separated by an empty line. 
>
> I would left as is to keep consistent with other comments, at least in this file

Fair enough.